staging/src/k8s.io/apiserver/pkg/server/hooks.go

```diff
@@ -232,13 +232,13 @@ func (h postStartHookHealthz) Name() string {
 	return h.name
 }
 
-var hookNotFinished = errors.New("not finished")
+var ErrHookNotFinished = errors.New("not finished")
```

Comment:
```suggestion
var errHookNotFinished = errors.New("not finished")
```

Reply:
I have modified it. Thanks

Reply:
Why make this error private to this package? Won't it be difficult to do error assertions on this error?

Reply:
It was the way it was, why do you want to export it?

Reply:
Thought maybe helpful when caller want to make error assertion when error returned.